### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by felddy**
# Lineage Pull Request: CONFLICT #

DANGER

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Clone the `lineage/skeleton` branch locally:

    ```console
    git clone --single-branch --branch lineage/skeleton git@github.com:cisagov/development-guide.git
    ```

1. Review each of the following files looking for [conflict-resolution markers](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts):

    ```console
    .pre-commit-config.yaml
    ```

1. Resolve each of the conflicts and `add` your changes to the branch:

    ```console
    git add .pre-commit-config.yaml 
    ```

1. Once all the changes have been added, `commit` and `push` the changes back to
GitHub:

    ```console
    git commit --message="Resolved Lineage conflicts."
    git push
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by mcdonnnj — APPROVED** [review]
:shipit:

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -60,25 +60,14 @@ repos:
...
       - id: black
-<<<<<<< HEAD
   # Disabling seed-isort-config in this project (development-guide) since it
   # doesn't identify docopt, et al. as third-party libraries.
   # - repo: https://github.com/asottile/seed-isort-config
-  #   rev: v2.1.0
+  #   rev: v2.1.1
   #   hooks:
   #     - id: seed-isort-config
-  - repo: https://github.com/pre-commit/mirrors-isort
```
Comment (jsf9k) [inline .pre-commit-config.yaml:69]: Why was this removed?  Is it no longer needed?

**Review by jsf9k — CHANGES_REQUESTED** [review]
It looks like a hook was removed here that perhaps should not have been.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (dav3r) [reply .pre-commit-config.yaml:69]: This removal is correct- see https://github.com/cisagov/skeleton-generic/commit/ce6658abc69d1444e51f45c20441c98e6b54c086 and https://github.com/cisagov/skeleton-generic/commit/2399f24622b1d6be92c494653d9308e9694c5496.

**Review by dav3r — APPROVED** [review]
LGTM! 🧶

File: .pre-commit-config.yaml (hunk 1 above)
Comment (felddy) [reply .pre-commit-config.yaml:69]: It was removed upstream: 
https://github.com/cisagov/skeleton-generic/commit/2399f24622b1d6be92c494653d9308e9694c5496#diff-82c3b36709926de3a1b27bea0d8258cc

Paging @mcdonnnj

Comment (jsf9k) [reply .pre-commit-config.yaml:69]: Ah, OK.  My ancient eyes mis-saw what they were looking at.  MY APOLOGIES SIR!!!!!

**Review by jsf9k — APPROVED** [review]
